content/cockpit-145.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Cockpit 145
+author: Martin Pitt
+date: 2017-07-07 16:00
+tags: cockpit, linux
+slug: cockpit-145
+category: release
+summary: Cockpit with Machines, Terminal, and Software Updates improvements
+comments: true
+---
+
+Cockpit is the [modern Linux admin interface](http://cockpit-project.org/). We release regularly.
+Here are the release notes from version 144 and 154.
+
+### Terminal size
+
+The Terminal now uses the full window size and dynamically resizes with it.
+Previously its size was fixed to 80x24 characters.
+
+![Full-window terminal](images/terminal-full-window.png)
+
+### Machines page reactiveness
+
+The Machines page now listens to libvirt events instead of reading the entire
+state of all virtual machines every ten seconds. This makes the page much more
+reactive to state changes and reduces the number of calls to `virsh`.
+
+### Delete VM functionality
+
+A "Delete" button/dialog was added to the Machines page. If the VM is still
+running, it will be force-stopped before. The dialog also offers you to remove
+the VM's disk images as well.
```

Comment:
...running, it will be force-stopped **first**.
The dialog also offers you **the choice** to remove